walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimeRange.java

```diff
@@ -49,7 +49,7 @@ public int compare(TimeRange a, TimeRange b) {
   private final int start;
   private final int duration;
 
-  private TimeRange(int start, int duration) {
+  protected TimeRange(int start, int duration) {
```

Comment:
Why change it?

Reply:
Needed to create TimeRange objs in my algorithm, but after doing tests I realized there are public functions! Will change back